src/reviews/reviews.controller.js

```diff
@@ -0,0 +1,45 @@
+const reviewModel = require('./reviews.model')
+/* 
+const controller = {
+}
+
+controller.reviews = (request, response) => {
+     response.status(200).json("Soy un test! OOOOOK")
+} */
+
+const createReview = async (req, res) => {
+     const review = await reviewModel.create(req.body);
+     res.status(201).json(review)
+};
+
+const getReview = async (req, res) => {
+     const review = await reviewModel.get(req.body.id);
```

Comment:
Cambiar body.id por params.reviewid:
```suggestion
     const review = await reviewModel.get(req.params.reviewid);
```